x-pack/test/fleet_cypress/config.ts

```diff
@@ -43,6 +43,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         `--xpack.fleet.enableExperimental=${JSON.stringify([
           'agentTamperProtectionEnabled',
           'subfeaturePrivileges',
+          'installedIntegrationsTabularUI',
```

Comment:
This is causing the `Fleet Cypress Tests #2 / Fleet startup Create policies should create agent policy` test to fail since it expects to interact with the card based UI. We should look at how to scope this flag just for the new test in `integrations_real.cy.ts`

Reply:
We probably will have to create a new config for those tests for example we did it for space awareness https://github.com/elastic/kibana/blob/080f29e27b61e63818a1b38daad7b8672ef95ffa/x-pack/platform/plugins/shared/fleet/cypress.config.space_awareness.ts

I have a separate issue to add cypress tests to that UI, we may want to delay test for your feature to that task https://github.com/elastic/kibana/issues/217212

Reply:
Oh good! I'll add a comment to that issue and link to the commit with this test